crates/cheatcodes/spec/src/vm.rs

```diff
@@ -2139,6 +2139,18 @@ interface Vm {
     /// Returns ENS namehash for provided string.
     #[cheatcode(group = Utilities)]
     function ensNamehash(string calldata name) external pure returns (bytes32);
+
+    /// Returns a random uint256 value.
+    #[cheatcode(group = Utilities)]
+    function randomUint() external returns (uint256);
+
+    /// Returns random uin256 value between the provided range (min..=max).
+    #[cheatcode(group = Utilities)]
+    function randomUint(uint256 min, uint256 max) external returns (uint256);
```

Comment:
Was looking into this from https://github.com/foundry-rs/forge-std/issues/562.  From this method + the test below, it seems there is no way to include 0 in the range when generating a random uint?

Reply:
they are both inclusive

Reply:
> they are both inclusive

Ok, so just need to update the docstring then.

IMO `randomUint` should also generate values of zero. The current test checks for `assertTrue(rand > 0)` which implies it does not

> I don't know why this exists if `bound` exists too.

This is something we should make sure to document since I suspect this will be a source of confusion, but my understanding from chatting with @mattsse:
- `bound` is for wrapping fuzz parameters, which are generated "intelligently" using strategies, into a given range
- `randomUint` generates a uniform distribution of numbers, not intended to replace fuzz parameters since it's purely random, and it for whenever you need an arbitrary random number for something